src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -1,16 +1,57 @@
 package com.google.codeu.servlets;
 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.codeu.data.Datastore;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+// Handles fetching and saving user data.
 @WebServlet("/about")
 public class AboutMeServlet extends HttpServlet {
+
+  private Datastore datastore;
+
+  @Override
+  public void init() {
+    datastore = new Datastore();
+  }
+
+  // Responds with the "about me" section for a particular user.
```

Comment:
You don't need to javadoc override functions. This is because the base function usually will have a javadoc that people can read. If you want to comment on specific implementation of this particular override function I would put the comments inside of the function instead.